src/foam/core/Window.js

```diff
@@ -53,6 +53,7 @@ foam.CLASS({
     'console',
     'debug',
     'delayed',
+    'discardStale',
```

Comment:
framed and idled need to come from the Window because they rely on methods provided by the Window (like requestanimationframe) and if you have multiple Windows then you need to get the correct one, but discardStale doesn't rely on these so doesn't need to be here.